modules/checklist/steps/setup-header.php

```diff
@@ -25,6 +26,14 @@ public function get_id() : string {
 		return self::STEP_ID;
 	}
 
+	public function is_visible() : bool {
```

Comment:
```suggestion
	public static function is_active() : bool {
```
Nitpick comment. But maybe consider calling it `is_active` to be consistent with other conditional loading throughout the code base.

Examples:
```
if ( $class_name::is_active() ) {
	$this->modules[ $module_name ] = $class_name::instance();
}
```

(Not a blocker).

Reply:
@louiswol94 the `static` for this method is more for a consistency sake, or do we have some other reasoning for this preference?

I don't mind changing the name...

though from what I see in the code-base, most of the time it's used in a context of a module, that can be "activated" / "deactivated", not sure our steps must follow in this case.

but again, when this goes public, it's public, so we need to consider wisely.. 

@TzviRabinovitch how do you feel about that? should we do it?

Reply:
we do not need a static function here
I think is_visible is more relevant here 
a module may be active or not 
a step is hidden or not